lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -229,7 +236,7 @@ def configIpBlock(self):
         print("************************* Waiting for IP changes to propagate")
         self._mobility_util.wait_for_changes()
 
-    def configUEDevice(self, num_ues, req_data=None, static_ips=None):
+    def configUEDevice(self, num_ues, req_data=None, static_ips=None, mock_pcrf=False):
```

Comment:
What is the difference between `self.mock_pcrf` and the argument `mock_pcrf` here? Is the argument actually needed?